dev-docs/faq.md

```diff
@@ -173,6 +171,24 @@ that [bid adapters resolve OpenRTB macros](/dev-docs/bidder-adaptor.html#resolve
 For historic reasons, Prebid will resolve the AUCTION_PRICE macro, but it will be after currency conversion and any bid adjustments.
 This differs from how OpenRTB defines this value as being the clearing price in the bid currency. Header Bidding is a first-price auction, the best candidate for “clearing price” is the original bid itself.
 
+## How does Prebid interact with the GAM yield group header bidding feature?
+
+Prebid welcomes the help from Google towards making header bidding easier for the community. Here's their [official blog post](https://blog.google/products/admanager/improved-header-bidding-support-in-google-ad-manager/). The creation of thousands of line items is clearly a hassle (or barrier) for publishers who want to utilize header bidding on the open web, so this feature is a great step in the right direction.
```

Comment:
In addition, the improved auction dynamics from no longer using price bucketing are likely to have beneficial effects on auction outcomes.